js/web/src/NinaTheme.js

```diff
@@ -37,6 +39,8 @@ export const NinaTheme = createTheme({
       primary: colors.grey,
     },
     greyLight: colors.greyLight,
+    connectionDotColorConnected: colors.connectionDotColorConnected,
```

Comment:
theme file should have reusable / color-descriptive values (green, lightGreen) but no biggie